src/app/app.component.html

```diff
@@ -1,5 +1,6 @@
 <app-header></app-header>
 <app-toolbar></app-toolbar>
+<app-food-counter></app-food-counter>
```

Comment:
Maybe when we style more this could go in the toolbar on the right side?

Reply:
Yeah that sounds good.